dotCMS/src/main/java/com/dotmarketing/quartz/job/ContentFromEmailJob.java

```diff
@@ -51,243 +49,232 @@
+									value = email.get("attachment6");
+							}else if(UtilMethods.isSet(attachment7FieldVarNm) && field.getVelocityVarName().equals(attachment7FieldVarNm)){
+								if(email.get("attachment7") != null)
+									value = email.get("attachment7");
+							}else if(UtilMethods.isSet(attachment8FieldVarNm) && field.getVelocityVarName().equals(attachment8FieldVarNm)){
+								if(email.get("attachment8") != null)
+									value = email.get("attachment8");
+							}else if(UtilMethods.isSet(attachment9FieldVarNm) && field.getVelocityVarName().equals(attachment9FieldVarNm)){
+								if(email.get("attachment9") != null)
+									value = email.get("attachment9");
+							}else if(UtilMethods.isSet(attachment10FieldVarNm) && field.getVelocityVarName().equals(attachment10FieldVarNm)){
+								if(email.get("attachment10") != null)
+									value = email.get("attachment10");
+							}
+
+							String typeField = field.getFieldType();
+
+							/* Validate if the field is read only, if so then check to see if it's a new contentlet
+							 * and set the structure field default value, otherwise do not set the new value.
+							 */
+							if (!typeField.equals(Field.FieldType.HIDDEN.toString()) &&
+									!typeField.equals(Field.FieldType.IMAGE.toString()) &&
+									!typeField.equals(Field.FieldType.FILE.toString()))
+							{
+								if(field.isReadOnly() && !InodeUtils.isSet(contentlet.getInode()))
+									value = field.getDefaultValue();
+								if (field.getFieldType().equals(Field.FieldType.WYSIWYG.toString())) {
+									//WYSIWYG workaround because the WYSIWYG includes a <br> even if the field was left blank by the user
+									//we have to check the value to leave it blank in that case.
+									if (value instanceof String && ((String)value).trim().toLowerCase().equals("<br>")) {
+										value = "";
+									}
+								}
+							}
+							if ((value != null || field.getFieldType().equals(Field.FieldType.BINARY.toString())) && APILocator.getFieldAPI().valueSettable(field)
+									&& !field.getFieldType().equals(Field.FieldType.HOST_OR_FOLDER.toString()))
+								try{
+									conAPI.setContentletProperty(contentlet, field, value);
+								}catch (Exception e) {
+									Logger.info(this, "Unable to set field " + field.getFieldName() + " to value " + value);
+									Logger.debug(this, "Unable to set field " + field.getFieldName() + " to value " + value, e);
+								}
+						}
+
+						contRel = retrieveRelationshipsData(contentlet,systemUser,email);
+
+						cats = new ArrayList<Category>();
+
+						Contentlet con = conAPI.checkin(contentlet, contRel,
+								cats, perAPI.getPermissions(contentlet, false, true), systemUser, false);
+
+						// When a large number of new mails with a number of replies come in.
+						// To relate the content based on lucene query, the new content created must be indexed.
+						conAPI.isInodeIndexed(con.getInode());
+					}
+				} catch (MessagingException e) {

```

Comment:
you could change this to MessageException | IOException | DotContentletValidationException ... e) {
    Logger.error(this, e.getMessage(), e);